src/components/Content/TaskManager/TaskForm/TaskForm.js

```diff
@@ -155,8 +156,9 @@ class TaskForm extends Component {
   focusDatePicker = () => this.datePickerRef.input.focus();
 
   render() {
-    const { taskForm } = this.props;
+    const { taskForm, autoComplete } = this.props;
     const { title, assigned_to, due_date_raw } = taskForm[this.id] || {};
+    console.log('suggestions', autoComplete)
```

Comment:
@Moe-Shoman Needed?  If you really want to keep an eye on this, we can keep it for now.

Reply:
No, we can do without